src/assets/styles/sass/styles.scss

```diff
@@ -0,0 +1,58 @@
+.project-card {
+    height: 200px;
+    width: 300px;
+    border: 1px solid rgb(213, 213, 213);
+    position: relative;
+
+    h3 {
+        text-align: center;
```

Comment:
Codacy found an issue: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/manual/descholar-ceo/portfolio/pullRequest?prid=5731031)